compiler/stmt_test.py

```diff
@@ -104,10 +104,16 @@ def testAugAssignBitAnd(self):
         foo &= 3
         print foo""")))
 
-  def testAugAssignUnsupportedOp(self):
-    expected = 'augmented assignment op not implemented'
-    self.assertRaisesRegexp(util.ParseError, expected,
-                            _ParseAndVisit, 'foo **= bar')
+  def testAugAssignPow(self):
+    self.assertEqual((0, '64\n'), _GrumpRun(textwrap.dedent("""\
+        foo = 8
+        foo **= 2
+        print foo""")))
+
+#  def testAugAssignUnsupportedOp(self):
```

Comment:
I'm not sure there are any more unsupported AugAssign operators, so you can just remove it.